services/ui-src/src/types/reports.ts

```diff
@@ -309,6 +310,10 @@ export interface MCPARReportMetadata extends ReportMetadata {
   dueDate: number;
   combinedData: boolean;
   programIsPCCM: Choice[];
+  newOrExistingProgram: Choice[];
```

Comment:
will this type cause any issues when the flag is off and the report doesn't save this key?